### PR TITLE
Added Ultra96 Skin Cancer AI

### PR DESCRIPTION
**Pull request by julianbartolone**
Signed-off-by: Julian Bartolone <julianbartolone@hotmail.com>

### CODE REVIEW

**Comment by sdrobertw** [comment]
@ric96 can you please review these projects? Pending your approval, I will merge.

Thanks.

File: _projects/view/alexa-and-ultra96/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Integrating Alexa & Ultra96
+permalink: "/projects/alexa-and-ultra96/"
+description: |-
+    Voice control using Alexa is popular for a number of applications. This project shows how to connect an Ultra96 (Arm & FPGA) with Alexa.
+images:
+  - alexa-and-ultra96.png
```
Comment (ric96) [inline _projects/view/alexa-and-ultra96/README.md]: the actual image name seems to be alexa-and-ultra96.PNG and not alexa-and-ultra96.png.
Please rename the image to .png from .PNG

**Comment by sdrobertw** [comment]
@julianbartolone let's see how it goes! Everything looks good. Hopefully we get a successful build. Will monitor your PR's more closely to get them merged quicker. This way, you can submit more projects. 

If I do not merge, DM me on Discord or send me a text, and I will do my best to get them in there ASAP.

**Comment by ric96** [comment]
@julianbartolone looks like there are more broken links to images. Can you go through and make sure the fike names are correct?
`Invalid image path specified: _projects/view/ultra-96-skin-cancer-ai/images/skincancerai.png (SyntaxError)`

**Pull request merged**